configure.in

```diff
@@ -58,6 +58,9 @@ SAVE_CFLAGS="$CFLAGS"
 AC_PROG_CC([$($PG_CONFIG --cc)])
 CFLAGS="$SAVE_CFLAGS"
 
+# Add the Postgres PKGINCLUDEDIR to the include search path
+CPPFLAGS="$CPPFLAGS -I$($PG_CONFIG --pkgincludedir)"
+
```

Comment:
Two things:
a) for the benefit of odd platforms^WOSX we include the generated configure script in git (as does postgres).
b) I'm a bit confused about having to play with cppflags that way. postgres' Makefile.global.in, included via pgxs.mk, included via citus'  Makefile.global.in has:
```
pkgincludedir := $(shell $(PG_CONFIG) --pkgincludedir)
...
includedir_server = $(pkgincludedir)/server
includedir_internal = $(pkgincludedir)/internal
...
CPPFLAGS = @CPPFLAGS@

ifdef PGXS
override CPPFLAGS := -I$(includedir_server) -I$(includedir_internal) $(CPPFLAGS)
else # not PGXS
```
which files aren't found without the above?  I'd wager it's failures in connection_test.c? Per #411 I think the actual issue there is that internal/ has mistakenly been added to the include path.